doc/modules/cross_validation.rst

```diff
@@ -650,6 +651,10 @@ Here is a visualization of the cross-validation behavior.
    :align: center
    :scale: 75%
 
+Similar to k-fold, the test sets from :class:`GroupKFold` will form a complete
+partition of all the data. Unlike k-fold, :class:`GroupKFold` is not randomized
+at all, whereas :class:`KFold` is randomized when ``shuffle=True``.
```

Comment:
To me, the distinction between :class:`KFold` and `"k-fold"` is not really clear. Can this be made more precise?

(To me, :class:`KFold` is the simplest strategy of k-folding.)

Reply:
Ah I thought they meant the same thing, I have amended to use :class:Kfold instead